xrpl4j-address-codec/src/test/java/org/xrpl/xrpl4j/codec/addresses/UnsignedByteArrayTest.java

```diff
@@ -41,4 +42,96 @@ public void lowerCaseOrUpperCase() {
     assertThat(UnsignedByteArray.fromHex("abcdef0123").hexValue()).isEqualTo("ABCDEF0123");
   }
 
+  @Test
+  public void simpleEmpty() {
+    assertThat(UnsignedByteArray.of(new byte[] {0}).empty()).isEqualTo(UnsignedByteArray.of(new byte[] {}));
+    assertThat(UnsignedByteArray.of(new byte[] {0, MAX_BYTE}).empty()).isEqualTo(UnsignedByteArray.of(new byte[] {}));
+    assertThat(UnsignedByteArray.of(new byte[] {MAX_BYTE, MAX_BYTE}).empty().length()).isEqualTo(0);
+    assertThat(
+      UnsignedByteArray.of(new byte[] {0, MAX_BYTE}).empty().equals(UnsignedByteArray.of(new byte[] {}))
+    ).isTrue();
+  }
+
+  @Test
+  public void simpleLength() {
```

Comment:
can we get rid of the "simple" prefix on all of these names? The fact that there is no "complicated" counterpart is a bit confusing lol